README.md

```diff
@@ -283,6 +283,7 @@ Use the "Table of Contents" menu on the top-left corner to explore the list.
 - [vite-plugin-load-css-module](https://github.com/jsonz1993/vite-plugin-load-css-module) - Support custom regexp of css modules and use vite internal css plugin, not just `.module.xxx`.
 - [vite-plugin-remain-exports](https://github.com/coconilu/vite-plugin-remain-exports) - Remaining exports from the entry file that imported by html.
 - [vite-plugin-iso-import](https://github.com/bluwy/vite-plugin-iso-import) - Import modules isomorphically in the client or server.
+- [vite-plugin-unocss-to-uni](https://github.com/zguolee/vite-plugin-unocss-to-uni) - A Vite plugin that supports [UnoCSS](https://github.com/unocss/unocss) in [uni-app](https://github.com/dcloudio/uni-app).
```

Comment:
## Checklist

- [ ] git: pull request template structure not broken
- [ ] git: pull request template not selected `type`
- [ ] git: pull request template not checked `Checklist`
- [ ] git: empty commit
- [ ] git: link not to repository
- [ ] git: link description contains a link to the author / third-party resource
- [x] git: link description no `for Vite` or `Vite plugin`
- [ ] git: link description no `Fast` or `Super-fast` etc
- [ ] git: link description short description short description (one sentence in 24 words, no expanded explanation)
- [ ] git: terms in the wrong formats (e.g. Typescript over TypeScript)
- [ ] git: broken PR
- [ ] git: no conflicts resolved
- [ ] git: doublicate
- [ ] time: not updated for more than 6 months
- [ ] category: not in its category
- [ ] lists: not the last on the list
- [ ] docs: no documentation
- [ ] docs: non-english documentation
- [ ] project: commercial product
- [ ] project: dead project
- [ ] project: does not accept donat
- [ ] site: no link to the site
- [ ] site: site with errors
- [ ] site: not actively using vuejs
- [ ] site: not original
> Follow the rules [CONTRIBUTING](https://github.com/vitejs/awesome-vite/blob/master/.github/contributing.md)

Reply:
OK, I have updated the readme.